src/components/index.js

```diff
@@ -1,9 +1,7 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
-import FlatmapVuer from "./FlatmapVuer.vue";
-import MultiFlatmapVuer from "./MultiFlatmapVuer.vue";
-  
-export {
-  FlatmapVuer,
-  MultiFlatmapVuer
-};
+import FlatmapVuer from './FlatmapVuer.vue'
+import MultiFlatmapVuer from './MultiFlatmapVuer.vue'
+import Tooltip from './Tooltip.vue'
```

Comment:
Does the tooltip get used externally now?

Reply:
Yes